examples/03-widgets/checkbox-widget.py

```diff
@@ -1,4 +1,5 @@
 """
+.. _checkbox_widget_example:
```

Comment:
I'm not sure if this line needs an empty line below it. We can wait and see how the documentation gets built in CI. (Or we could build the current state locally, but I'm not patient enough for that.)